src/components/PokedexView/PokedexView.js

```diff
@@ -54,17 +25,15 @@ function PokedexView( { generation } ) {
 
 			<div className="pokedex-view">
 				{
-					pokemons.map( ( p, i ) => {
+					pokemons.map( ( pokemon ) => {
```

Comment:
Seems like it should be a separated component, and only this new component should use the usePokemons and the useGeneration hooks (in order to prevent multiple re-renders of the rest of the components).

Reply:
I moved them to the App